src/utils/queryStringify.ts

```diff
@@ -0,0 +1,6 @@
+export function queryStringify(data: Record<string, any>) {
+  const keys = Object.keys(data);
+  return keys.reduce((result, key, index) => {
+    return `${result}${key}=${data[key]}${index < keys.length - 1 ? '&' : ''}`;
+  }, '?');
+}
```

Comment:
можно записать попроще:
```suggestion
export function queryStringify(data: Record<string, string>): string {
  return "?" + Object
      .entries(data)
      .map(([key, value]) => `${key}=${value}`)
      .join("&")
}
```